src/common/SPELLS/TALENTS/WARRIOR.js

```diff
@@ -22,6 +22,7 @@ export default {
   TITANIC_MIGHT_TALENT: { id: 202612, name: "Titanic Might", icon: "ability_warrior_colossussmash" },
   DEADLY_CALM_TALENT: { id: 227266, name: "Deadly Calm", icon: "achievement_boss_kingymiron" },
   IN_FOR_THE_KILL_TALENT: { id: 248621, name: "In For The Kill", icon: "ability_blackhand_marked4death" },
+  IN_FOR_THE_KILL_TALENT_BUFF: { id: 248622, name: "In For The Kill", icon: "ability_blackhand_marked4death" },
```

Comment:
The talent files are auto generated, so this would be deleted the next time they are generated. You should instead add it to src/common/SPELLS//WARRIOR.js.